client/my-sites/stats/features/modules/stats-utm/stats-module-utm.jsx

```diff
@@ -241,11 +216,6 @@ const StatsModuleUTM = ( {
 										</div>
 									}
 								/>
-								{ showFooterWithDownloads && (
-									<div className="stats-module__footer-actions stats-module__footer-actions--summary">
-										<UTMExportButton data={ data } fileName={ fileNameForExport } />
-									</div>
-								) }
```

Comment:
Removing the footer here means the surrounding fragment `<>…</>` should no longer be needed.

Reply:
Good catch! I just got rid of the fragment.